### ISSUE TITLE
Is there a light-model version?

### ISSUE THREAD

**Issue by erwangccc** [Diananini/LD-ConGR-CVPR2022#3]
Maybe current model is unfriendly to edge devices, is there a light-model version?
Thanks

**Comment by Diananini** [comment]
The model in this paper is not designed for edge devices. Maybe we will try to propose a lightweight gesture recognition model in the future.

**Comment by erwangccc** [comment]
Thanks for your quick response. Looking forward to your future great work.
Do you have any recommended light gesture recognition models based on RGB-D data? Thanks in advance.

**Comment by Diananini** [comment]
As far as I know, there is no lightweight model specifically for gesture recognition. Maybe you can try lightweight models proposed for action recognition, such as X3D.

**Comment by erwangccc** [comment]
Thank you.
I want to confirm that, the paper says the gesture(hand) region is predicted and it will enlarge the hand region, so you will resize all 32/16 frames to input size 112x112, right? But we do resize directly, the hand will deform, is it OK?

**Comment by Diananini** [comment]
Resize is used in both training and testing, and there is deformation in both training and testing. The key is that the distribution of test data is consistent with the training data, rather than one with deformation and the other without.

**Comment by erwangccc** [comment]
Thank you. Maybe padding by zero is ok,right?

**Comment by Diananini** [comment]
Yes, you can try it.
